Source/Renderer/Context.js

```diff
@@ -315,6 +316,7 @@ function Context(canvas, options) {
     "WEBKIT_WEBGL_compressed_texture_pvrtc",
   ]);
   this._etc1 = !!getExtension(gl, ["WEBGL_compressed_texture_etc1"]);
+  loadKTX2.setKTX2SupportedFormats(this._etc1, this._s3tc, this._pvrtc);
```

Comment:
It's unfortunate that `loadKTX2` needs to be notified of the supported formats in this way but probably required since imagery layers don't have access to the context. I would just add a comment about that.